ContactManagerApp/Pages/Index.cshtml.cs

```diff
@@ -61,24 +66,20 @@ private async Task LoadSampleData(CancellationToken cancellationToken)
             await _db.SaveChangesAsync(cancellationToken);
             return;
         }
-
             Contacts = await _db.ToListAsync(cancellationToken);
-
     }
 
-    /*
-    [BindProperty, Required, DisplayName("Person's Email")]
-    public string? Email { get; set; }
-
-    [BindProperty, Required, DisplayName("Person's Address")]
-    public string? Address { get; set; } */
-
     public async Task<IActionResult> OnPost(CancellationToken cancellationToken)
     {
         if (ModelState.IsValid)
         {
-            Contact contact = new() { FirstName = FirstName!, LastName = LastName!,
```

Comment:
`FirstName = FirstName!` isch bad practice, entweder s field isch nullable oder es isch nöd nullable. Da du s field in de Contact class scho non null gmacht hesch, wärs nur logisch zum das property binding au non null mache würsch.

Reply:
also entweder isch es uf beidne site string? oder es isch uf beidne site string aber im binding string? und in de klass string isch chli weird